spec/models/destroy_all_deletes_last_seven_days_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'Project manager creates TODO' do
```

Comment:
model specs use match the model they're spec'ing (eg todo_spec.rb), use the describe syntax and test specific methods for the model. this is a feature spec